lingui.config.js

```diff
@@ -8,10 +8,6 @@ module.exports = {
   fallbackLocales: {
     default: nextConfig.i18n.defaultLocale, // en
   },
```

Comment:
I think you also can drop this. Lingui will always fallback to source locale while compiling